pkg/cli/requestproject/request_project.go

```diff
@@ -67,7 +67,7 @@ to build a new example application in Ruby. Or use kubectl to deploy a simple Ku
 
 To switch to this project and start adding applications, use:
 
-    oc project %[2]s
+    oc project %[1]s
```

Comment:
make sense for `fmt.Fprintf(o.Out, requestProjectSwitchProjectOutput, o.ProjectName, o.Server)`